backend/cmd/migrate/migrate.go

```diff
@@ -2,43 +2,44 @@ package main
 
 import (
 	"database/sql"
-	"errors"
-	"flag"
 	"fmt"
 	"os"
+	"path/filepath"
 
 	"github.com/golang-migrate/migrate/v4"
 	"github.com/golang-migrate/migrate/v4/database/postgres"
 	_ "github.com/golang-migrate/migrate/v4/source/file"
 	_ "github.com/lib/pq"
+	"github.com/uber-go/tally"
+	"go.uber.org/zap"
+
+	"github.com/lyft/clutch/backend/gateway"
+	clutchpg "github.com/lyft/clutch/backend/service/db/postgres"
 )
 
-func run(args []string) error {
-	flags := flag.NewFlagSet(args[0], flag.ExitOnError)
-	var (
-		migrationDir = flags.String("migrations", "./migrations", "Directory with sql migrations.")
-		// TODO: Read from the gateway config to create the connection.
-		//config       = flags.String("config", "../backend/clutch-config.yaml", "config yaml to parse for db connection")
-		connection = flags.String("connection", "", "Connection string for the database.")
-	)
+func run() error {
+	f := gateway.ParseFlags()
```

Comment:
A dangerous side effect of running this without args is the usage of ones `clutch-config.yaml` which could be their production configuration. We should probably output what datastore we are modifying and ask the user to confirm before proceeding.